src/sections/Feature.vue

```diff
@@ -43,6 +43,13 @@
           >
             <i>{{ season }} applications are now open!</i>
           </p>
+          <p
+            v-else
+            class="secondary"
+          >
+            <i>Unfortunately we are not currently accepting new applications, but in the
+              meantime, keep an eye on this website for updates!</i>
```

Comment:
how about a simple concise:

```suggestion
            <i>We open applications every semester - follow us on social for updates!</i>
```

or similar?

Reply:
* positive (not "we are not...")
* stronger CTA ("we *do* x... *follow* us for updates!")
* specific towards recruitement, doesn't take away from the website (not as "go away nothing is happening now")

Reply:
I like that one

Reply:
Sounds good!